gen.go

```diff
@@ -174,3 +181,50 @@ func checkoutTemplateRef(wt *git.Worktree, templateRef string) error {
 
 	return fmt.Errorf("failed to checkout %s: not a valid hash, branch, or tag", templateRef)
 }
+
+// checkNecessaryBinaries calls checkApp and returns err
+// Tries to find excutable files on user's machine and return an ErrorOrNil
+// with links to install if the dependencies' excutables are not found
+func checkNecessaryBinaries() error {
+
+	var result *multierror.Error
+
+	if err := checkApp("go version"); err != nil {
+		err = errors.New("go is not installed, to install it see https://go.dev/doc/install")
+		result = multierror.Append(result, err)
+	}
+
+	if err := checkApp("docker version"); err != nil {
+		err = errors.New("docker is not installed, to install it see https://docs.docker.com/get-docker")
+		result = multierror.Append(result, err)
+	}
+
+	if err := checkApp("node -v"); err != nil {
+		err = errors.New("node is not installed, to install it see  https://docs.npmjs.com/downloading-and-installing-node-js-and-npm")
+		result = multierror.Append(result, err)
+	}
+
+	if err := checkApp("psql -V"); err != nil {
+		err = errors.New("psql is not installed, to install it see https://www.postgresql.org/download")
+		result = multierror.Append(result, err)
+	}
+
+	if err := checkApp("task --version"); err != nil {
+		err = errors.New("task is not installed, to install it see  https://taskfile.dev/installation")
+		result = multierror.Append(result, err)
+	}
+	return result.ErrorOrNil()
+}
+
+// checkApp looks for executables to run on user's local machine
+// and return nil if already installed and err otherwise
+func checkApp(args string) error {
+	cmd := exec.Command(args)
+	if errors.Is(cmd.Err, exec.ErrDot) {
+		cmd.Err = nil
+	}
```

Comment:
I think we shouldn't include this part. They made the library stop resolving local binaries for security purposes. The apps we're looking for really should be in the path. So I'd just delete these 3 lines.